packages/arb-token-bridge-ui/src/pages/api/arbstats.ts

```diff
@@ -0,0 +1,87 @@
+// Connects to Arbiscan and scrapes some nice info (currently only TPS) which we can show on our UI as well
+
+import axios from 'axios'
+import { load } from 'cheerio'
```

Comment:
we don't have cheerio on the `package.json` but it's a dep of a dep of a dep of [...] and hence is in the `yarn.lock`, do you want to install it?